app/code/community/Bolt/Boltpay/Helper/Api.php

```diff
@@ -990,7 +990,7 @@ public function getShippingAndTaxEstimate( $quote )
                 );
             }
 
-            $adjustedShippingAmount = $this->getAdjustedShippingAmount($originalDiscountedSubtotal, $quote);
```

Comment:
@YevhenShBolt we may need to be careful here. This code has been reviewed and tweaked several times and I'm worried about removing it. I believe that the case where it is needed is when a discount is applied to the subtotal based on a shipping method. For example, "$5 off when you choose Next Day Air".